src/Compiler/Checking/InfoReader.fs

```diff
@@ -561,8 +564,15 @@ type InfoReader(g: TcGlobals, amap: Import.ImportMap) as this =
     /// Filter the overrides of methods or properties, either keeping the overrides or keeping the dispatch slots.
     static let FilterOverrides findFlag (isVirt:'a->bool, isNewSlot, isDefiniteOverride, isFinal, equivSigs, nmf:'a->string) items = 
         let equivVirts x y = isVirt x && isVirt y && equivSigs x y
+        let filterDifiniteOverrides x = not (isDefiniteOverride x)
```

Comment:
I'd either add the filter into the function

```suggestion
        let filterDifiniteOverrides = List.filter (not << isDefiniteOverride)
```

Or remove it from the name

```suggestion
        let isNotDifiniteOverride x = not (isDefiniteOverride x)
```

Reply:
Fair, let me change it